packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -969,6 +1083,33 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
             bytes memory
         )
     {
+        uint256 messageValue = _nextMessageContext.ovmCALLVALUE;
+        // If there is value in this message, we need to transfer the ETH over before switching contexts.
+        if (messageValue > 0) {
+            // Handle out-of-intrinsic gas consistent with EVM behavior -- the subcall "appears to revert"
+            if (gasleft() < CALL_WITH_VALUE_INTRINSIC_GAS) {
```

Comment:
Requesting that we add links to the corresponding parts of Geth that confirm this is the standard behavior here.

Reply:
@smartcontracts can you help me find a link to the geth code on this one?  I can't track it down.

Reply:
I gotchu

Reply:
https://github.com/ethereum/go-ethereum/blob/c503f98f6d5e80e079c1d8a3601d188af2a899da/core/vm/interpreter.go#L268-L273

Reply:
I *think* this is correct but a sanity check would be appreciated.

Reply:
seems right, value should be a part of `dynamicGas`

Reply:
added